### PR TITLE
Label encoder persistent

### PR DESCRIPTION
**Pull request by paulzierep**
Using the h5 model dump and load to store the encoding of the label encoder

### CODE REVIEW

**Comment by bgruening** [comment]
You need to rebase here.

File: tools/sklearn/label_encoder.xml [hunk 1]
```diff
@@ -11,29 +11,59 @@
             --inputs '$inputs'
             --infile '$infile'
             --outfile '$outfile'
```
Comment (bgruening) [inline tools/sklearn/label_encoder.xml:13]: bump tool version?

**Comment by bgruening** [comment]
@anuprulez can you have a look here please?

**Comment by paulzierep** [comment]
Dropped for the clean version: https://github.com/bgruening/galaxytools/pull/1324

**Pull request closed without merging**